.changeset/fifty-teachers-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+'steiger': minor
```

Comment:
suggestion: let's make this a patch change. For one, it technically fixes something that should've been working all along, for two, I have larger plans for the next minor version, so I'd prefer to push it out as a patch change
```suggestion
'steiger': patch
```